src/components/ChatWidget/index.tsx

```
@@ -0,0 +1,214 @@
+  console.log(hostRef?.current?.shadowRoot, ">>shadowRoot");
+  const {
+    isSecondaryDrawerOpen,
+    secondaryDrawerContent,
+    toggleSecondaryDrawer,
+  } = useSecondaryDrawerController({
+    isMobile: true,
+    shadowRoot: hostRef.current?.shadowRoot,
+  });
+
+  const messages_map = useMemo(() => {
+    let _map = new Map();
```

ESLint

[error] '_map' is never reassigned. Use 'const' instead.

(prefer-const)